src/sbt-test/sbt-tpolecat/scalacOptions/build.sbt

```diff
@@ -244,3 +244,51 @@ TaskKey[Unit]("checkReleaseMode") := {
 
   assertEquals(actualOptions, expectedOptions)
 }
+
+TaskKey[Unit]("checkConsoleScalacOptions") := {
+  val shouldBeMissing: Set[String] = (ScalacOptions.privateWarnUnusedOptions ++
+    ScalacOptions.warnUnusedOptions ++
+    ScalacOptions.fatalWarningOptions +
+    ScalacOptions.privateWarnDeadCode +
+    ScalacOptions.warnDeadCode).flatMap(_.tokens).toSet
```

Comment:
GitHub displays this change wrongly. It is 

```
  val shouldBeMissing = ScalacOptions.defaultConsoleExclude.flatMap(_.tokens).toSet
```

see [here](https://github.com/DavidGregory084/sbt-tpolecat/pull/66/commits/e0aa717b375eb85f29e6a5bb1214f8715ef10be5)